seqr/views/utils/terra_api_utils.py

```diff
@@ -103,8 +103,14 @@ def _safe_get_social(user):
     if not google_auth_enabled() or not hasattr(user, 'social_auth'):
         return None
 
-    social = user.social_auth.filter(provider=SOCIAL_AUTH_PROVIDER)
-    return social.first() if social else None
+    social_auth = user.social_auth.filter(provider=SOCIAL_AUTH_PROVIDER)
+    if not social_auth:
+        return None
+
+    social = social_auth.first()
+    if isinstance(social.extra_data, str):  # JSONField extra_data is returned as a string
```

Comment:
is this actually a change we expect, or is this due to us using an older version of social auth with a newer version of django? If we bump the social auth version do we see a fix for this?

Reply:
Ah I think you were right in that the social-auth version needed to be bumped, additionally the fixtures needed extra_data to be defined as JSON dicts instead of strings.

Even if I changed the fixture strings to dicts without the social auth version bump, `extra_data` was returned as a string, so upgrading social auth is necessary. The json.loads() stuff I added is no longer necessary.